testing/src/main/java/org/oppia/testing/FakeCrashLogger.kt

```diff
@@ -0,0 +1,30 @@
+package org.oppia.testing
+
+import java.lang.Exception
+import org.oppia.testing.TestLogReportingModule.Companion.exceptionList
```

Comment:
We should never use static mutable state. If you want to share state across multiple components that are injecting this logger, you can define the exception list as a field of the class and mark the class as ``@Singleton`` so that its instance is retained across injections. You'll need to make this an injectable class for that to work, e.g.:

```kotlin
@Singleton
class FakeCrashLogger @Inject constructor() : Crash Logger {
```